### PR TITLE
 Add request path authenticator docs

### PR DESCRIPTION
**Pull request by DMHP**
Add request path authenticator docs

### CODE REVIEW

File: en/docs/guides/oauth-request-path.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+
+```tab="Request Format"
+    curl -v -X POST --basic -u <CLIENT_ID>:<CLIENT_SECRET> -H "Content-Type: application/x-www-form-urlencoded;charset=UTF-8" -k -d "grant_type=password&username=USERNAME&password=PASSWORD" https://localhost:9443/oauth2/token
+```
+
+```tab="Response Format"
+{"token_type":"Bearer","expires_in":2655,"refresh_token":"2f03de95b8e196f78c94d07c23c9ef0a","access_token":"7ee57bc28a3336ccb7818b499941e4e4"}
+```
+
+2. Send a cURL request using the access token you received as a response for step1 in the authorization header, to the token endpoint. Replace the <CLIENT_ID> and <REDIRECT_URI> tags with the relevant values.
```
Comment (sherenewso2) [inline en/docs/guides/oauth-request-path.md:38]: `<CLIENT_ID>` and `<REDIRECT_URI>` should be in monospace

File: en/docs/guides/oauth-request-path.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+
+```tab="Request Format"
+curl -v -X POST -H "Authorization: Bearer 7ee57bc28a3336ccb7818b499941e4e4" -H "Content-Type: application/x-www-form-urlencoded;charset=UTF-8" -k -d "response_type=code&client_id=<CLIENT_ID>&redirect_uri=<REDIRECT_URI>&scope=openid"  https://localhost:9443/oauth2/authorize
+```
+
+```tab="Response Format"
+Location: https://curl-app/callback?code=37c79c505960e90d5b25f62ce760c98c&session_state=6d1a72e0f3f6392d6648ec5e6ed0
+```
+
+3. Use the following cURL command to get an access token using the authorization code received in step2. Replace the <CLIENT_ID>:<CLIENT_SECRET>,<CLIENT_ID> and <REDIRECT_URI> tags with the relevant values.
```
Comment (sherenewso2) [inline en/docs/guides/oauth-request-path.md:48]: monspace text like <CLIENT_ID> and <REDIRECT_URI>

File: en/docs/guides/oauth-request-path.md [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+curl -v -X POST --basic -u <CLIENT_ID>:<CLIENT_SECRET>  -H "Content-Type: application/x-www-form-urlencoded;charset=UTF-8" -k -d "grant_type=authorization_code&client_id=<CLIENT_ID>&redirect_uri=<REDIRECT_URI>&code=37c79c505960e90d5b25f62ce760c98c&scope=openid" https://localhost:9443/oauth2/token
+```
+
+```tab="Response Format"
+{ "scope":"openid", "token_type":"Bearer", "expires_in":3600, "refresh_token":"70f202ca2e4ecf571d0b6d2e49af8f3a", "id_token":"eyJhbGciOiJSUzI1NiJ9.eyJhdXRoX3RpbWUiOjE0NjA0NTkzMTYsImV4cCI6MTQ2MDQ2MjkxNiwic3ViIjoiYWRtaW4iLCJhenAiOiJlN2VrQldVTVBITnFTNU5WQmhxNGhmNWZqMkVhIiwiYXRfaGFzaCI6IkhCWFVKQW50LWFMV3JxQlZJcTFoV2ciLCJhdWQiOlsiZTdla0JXVU1QSE5xUzVOVkJocTRoZjVmajJFYSJdLCJpc3MiOiJodHRwczpcL1wvbG9jYWxob3N0Ojk0NDNcL29hdXRoMlwvdG9rZW4iLCJpYXQiOjE0NjA0NTkzMTZ9.PiqVn7B2vuICHmodnn9udjQrvGqRR-PZr-M8x8Xijg0bnAvzXY4hxqZ5luaLitBH2IgQ5p0Rh_gjPI7TWcQA7AK3iBCp7c29QY78hSSqt38_iG5bC0MYWoluH-jg5f3iyJ3aQ-DPAZexCXxEv65RPF5EDNfhA0fUFcsu79cb89k", "access_token":"7d6c01fb6bfaca22f01d9a24219cce45" }
+```
+
+
+
+!!! troubleshoot
```
Comment (sherenewso2) [inline en/docs/guides/oauth-request-path.md:60]: !!! tip "Troubleshooting Tip"

File: en/docs/guides/oauth-request-path.md [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+        for the above step 9 will be as follows:
+    
+        ``` 
+        Location: https://localhost:9443/authenticationendpoint/oauth2_consent.do?loggedInUser=admin&application=plagroundapp&scope=openid&sessionDataKeyConsent=a14f4a5d-16bb-4e47-9c53-5eacee9828f2&spQueryParams=
+        ```
+    
+        This is the consent page which will require the user interaction to
+        either approve or deny the authorization request. You can go to the
+        above consent page url via a web browser and provide approve or
+        deny.Then the url will be redirected to
```
Comment (sherenewso2) [inline en/docs/guides/oauth-request-path.md:71]: The consent page URL given in the response is the consent page that requires user interaction to either approve or deny the authorization request. You can access the consent page URL via a web browser and either approve or deny consent. The URL will then be redirected to the following page.

File: en/docs/guides/oauth-request-path.md [hunk 5]
```diff
@@ -0,0 +1,80 @@
...
+
+```tab="Response Format"
+{ "scope":"openid", "token_type":"Bearer", "expires_in":3600, "refresh_token":"70f202ca2e4ecf571d0b6d2e49af8f3a", "id_token":"eyJhbGciOiJSUzI1NiJ9.eyJhdXRoX3RpbWUiOjE0NjA0NTkzMTYsImV4cCI6MTQ2MDQ2MjkxNiwic3ViIjoiYWRtaW4iLCJhenAiOiJlN2VrQldVTVBITnFTNU5WQmhxNGhmNWZqMkVhIiwiYXRfaGFzaCI6IkhCWFVKQW50LWFMV3JxQlZJcTFoV2ciLCJhdWQiOlsiZTdla0JXVU1QSE5xUzVOVkJocTRoZjVmajJFYSJdLCJpc3MiOiJodHRwczpcL1wvbG9jYWxob3N0Ojk0NDNcL29hdXRoMlwvdG9rZW4iLCJpYXQiOjE0NjA0NTkzMTZ9.PiqVn7B2vuICHmodnn9udjQrvGqRR-PZr-M8x8Xijg0bnAvzXY4hxqZ5luaLitBH2IgQ5p0Rh_gjPI7TWcQA7AK3iBCp7c29QY78hSSqt38_iG5bC0MYWoluH-jg5f3iyJ3aQ-DPAZexCXxEv65RPF5EDNfhA0fUFcsu79cb89k", "access_token":"7d6c01fb6bfaca22f01d9a24219cce45" }
+```
+
+
+
+!!! troubleshoot  
+        If you haven't disabled the consent as pointed above, then the response
+        for the above step 9 will be as follows:
```
Comment (sherenewso2) [inline en/docs/guides/oauth-request-path.md:62]: If you have not disabled consent, the response will be as follows.

File: en/docs/quick-starts/basic-auth-request-path-sample.md [hunk 6]
```diff
@@ -0,0 +1,59 @@
+# Enable Authentication with Basic Auth Request Path Authenticator
+
+This page guides you through enabling authentication for an OpenID Connect web application using a **sample application** called Playground2.
```
Comment (sherenewso2) [inline en/docs/quick-starts/basic-auth-request-path-sample.md:3]: ...enabling basic authentication using a request path authenticator...

File: en/docs/quick-starts/basic-auth-request-path-sample.md [hunk 7]
```diff
@@ -0,0 +1,59 @@
...
+               `                               http://wso2is.local:8080/playground2/oauth2client                             `
+           
+   -   **Access Token Endpoint** :
+               `               https://localhost:9443/oauth2/token              `
+           
+   -   **Authorize Endpoint:**
+               `                               https://localhost:9443/oauth2/authorize?sectoken=                              <sec_token>              `
+
+!!! info 
+    The sectoken in the Authorize Endpoint will be the `username:password` in Base64
```
Comment (sherenewso2) [inline en/docs/quick-starts/basic-auth-request-path-sample.md:40]: `sectoken` 

Authorize Endpoint --> authorization endpoint

File: en/docs/quick-starts/oauth-request-path-sample.md [hunk 8]
```diff
@@ -0,0 +1,59 @@
+# Enable Authentication with OAuth Request Path Authenticator
+
+This page guides you through enabling authentication for an OpenID Connect web application using a **sample application** called Playground2.
```
Comment (sherenewso2) [inline en/docs/quick-starts/oauth-request-path-sample.md:3]: ...enabling request path authentication...

File: en/docs/quick-starts/oauth-request-path-sample.md [hunk 9]
```diff
@@ -0,0 +1,59 @@
...
+ -   **Resource Owner User Name:** User Name
+ -   **Resource Owner Password:** Password of the user
+ -   **Callback URL:**
+     `                               http://wso2is.local:8080/playground2/oauth2client                             `
+ 
+ -   **Access Token Endpoint:**
+     `                               https://localhost:9443/oauth2/token                             `
+ 
+!!! info
+    Once you receive the access token, you can use the following
```
Comment (sherenewso2) [inline en/docs/quick-starts/oauth-request-path-sample.md:40]: use the following to...??

File: en/mkdocs.yml [hunk 10]
```diff
@@ -132,7 +135,6 @@ nav:
           - 'Discover OpenID Connect Provider': guides/login/oidc-discovery.md
           - 'Obtain Key Set Using JWKS': guides/login/using-jwks.md
           - 'Validate JWT based on JWKS': guides/login/validate-jwt-using-jwks.md
-          - 'Use advanced parameters in authentication requests': guides/login/advanced-parameters-in-authentication.md
```
Comment (sherenewso2) [inline en/mkdocs.yml:135]: are we removing this topic?

File: en/docs/fragments/register-an-application-request-path.md [hunk 11]
```diff
@@ -0,0 +1,31 @@
...
+
+8.	Click **Next**. 
+
+9.	Check the configurations and click **Finish**. 
+
+10.	Navigate to the **Access** tab. 
+
+11.	Select all the options given under **Allowed grant type**.
+
+12. Navigate to the **Sign-on Method** tab.
```
[the remark this answers is not in the document]
Comment (DMHP) [reply en/docs/fragments/register-an-application-request-path.md:27]: Then we need to maintain 14.Click **Update**. in a complete new fragment right?

File: en/mkdocs.yml (hunk 10 above)
Comment (DMHP) [reply en/mkdocs.yml:135]: It is duplicated.

File: en/docs/fragments/register-an-application-request-path.md (hunk 11 above)
[the remark this answers is not in the document]
Comment (sherenewso2) [reply en/docs/fragments/register-an-application-request-path.md:27]: we could put 13. and 14 in to the original document where the fragment is pulled into

in this case in the file quick-starts/oauth-request-path-sample.md, we can add steps 13 and 14 right after the {!fragments!} include ?

File: en/mkdocs.yml [hunk 12]
```diff
@@ -160,36 +162,67 @@ nav:
...
-      - 'Manage Users':
+      - 'Identity Lifecycles':
+        - 'Onboard Users':
+          - 'Admin Creation workflow': guides/onboard/admin-creation-workflow.md
+          - 'Invitation workflow': guides/user-mgt/ask-password.md
+          - 'User self-registration workflow': guides/onboard/user-self-registration.md
+          - 'Just in time user provisioning workflow': guides/onboard/just-in-time.md
+          - 'Bulk Import Users': guides/user-mgt/import-users.md
+          - 'Outbound Provisioning': guides/onboard/outbound-provisioning.md
+          - 'Approval workflow': guides/onboard/approval-workflow.md
```
Comment (sherenewso2) [inline en/mkdocs.yml:179]: Let's use Title Case for these

File: en/mkdocs.yml [hunk 13]
```diff
@@ -160,36 +162,67 @@ nav:
...
+          - 'Disable Accounts': guides/user-mgt/enable-account.md
+      - 'Password Management':
+        - 'Password Policies': guides/user-mgt/password-policies.md
+        - 'Admin-initiated password reset': guides/user-mgt/forced-password-reset.md
+        - 'Password recovery using email verification': guides/user-mgt/recover-password.md
+        - 'Password recovery using challenge question': guides/user-mgt/challenge-question.md
+        - 'ReCaptcha on invalid password attempts': guides/user-mgt/recaptcha-invalid-password-attempts.md
+        - 'ReCaptcha on invalid challenge question attempt': guides/user-mgt/recaptcha-challenge-question-attempts.md
+      - 'Federated Account Linking':
+        - 'Automatic Account Linking during JIT': guides/automatic-account-linking.md
```
Comment (sherenewso2) [inline en/mkdocs.yml:206]: Title Case

File: en/mkdocs.yml [hunk 14]
```diff
@@ -160,36 +162,67 @@ nav:
...
+        - 'Admin Intiated Password Reset': guides/admin-intiated-password-reset.md
```
Comment (sherenewso2) [inline en/mkdocs.yml:207]: this topic is repeated under "Password Management"

File: en/mkdocs.yml [hunk 15]
```diff
@@ -160,36 +162,67 @@ nav:
...
+          - 'Lock/unlock Accounts': guides/user-mgt/lock-account.md
+          - 'Suspend Accounts': guides/states/suspend-accounts.md
+          - 'Disable Accounts': guides/user-mgt/enable-account.md
+      - 'Password Management':
+        - 'Password Policies': guides/user-mgt/password-policies.md
+        - 'Admin-initiated password reset': guides/user-mgt/forced-password-reset.md
+        - 'Password recovery using email verification': guides/user-mgt/recover-password.md
+        - 'Password recovery using challenge question': guides/user-mgt/challenge-question.md
+        - 'ReCaptcha on invalid password attempts': guides/user-mgt/recaptcha-invalid-password-attempts.md
+        - 'ReCaptcha on invalid challenge question attempt': guides/user-mgt/recaptcha-challenge-question-attempts.md
```
Comment (sherenewso2) [inline en/mkdocs.yml:204]: Title Case

File: en/mkdocs.yml [hunk 16]
```diff
@@ -160,36 +162,67 @@ nav:
...
+          - 'Microsoft Azure AD': guides/onboard/asure.md
+          - 'Google Directory': guides/identity-provisioning/outbound-provisioning-with-google.md
+          - 'Salesforce': guides/identity-provisioning/outbound-provisioning-with-salesforce.md
+          - 'Role Based Provisioning': guides/identity-provisioning/role-based-provisioning.md
+          - 'Rule Based Provisioning': guides/identity-provisioning/rule-based-provisioning.md
+        - 'Manage Userstores':
+          - 'JDBC as the Primary Userstore': guides/userstore/jdbc-us.md
+          - 'Active Directory as a Secondary Userstore': guides/userstore/ad-us.md
+          - 'LDAP as a Secondary Userstore': guides/userstore/ldap-us.md
+          - 'Plug a Custom Userstore': guides/userstore/custom-us.md
```
Comment (sherenewso2) [inline en/mkdocs.yml:193]: Userstore --> User Store

File: en/mkdocs.yml [hunk 17]
```diff
@@ -160,36 +162,67 @@ nav:
...
+      - 'Password Management':
+        - 'Password Policies': guides/password-mgt/password-policies.md
+        - 'Admin-initiated password reset': guides/password-mgt/forced-password-reset.md
+        - 'Password recovery using email verification': guides/password-mgt/recover-password.md
+        - 'Password recovery using challenge question': guides/password-mgt/challenge-question.md
+        - 'ReCaptcha on invalid password attempts': guides/password-mgt/recaptcha-invalid-password-attempts.md
+        - 'ReCaptcha on invalid challenge question attempt': guides/password-mgt/recaptcha-challenge-question-attempts.md
+      - 'Federated Account Linking':
+        - 'Automatic Account Linking during JIT': guides/account-linking/automatic-account-linking.md
+        - 'Admin Intiated Password Reset': guides/account-linking/admin-intiated-password-reset.md
```
Comment (sherenewso2) [inline en/mkdocs.yml:207]: This topic has been repeated under "Password Management"

File: en/mkdocs.yml [hunk 18]
```diff
@@ -160,36 +162,67 @@ nav:
...
+          - 'Lock/unlock Accounts': guides/account-states/lock-account.md
+          - 'Suspend Accounts': guides/account-states/suspend-accounts.md
+          - 'Disable Accounts': guides/account-states/enable-account.md
+      - 'Password Management':
+        - 'Password Policies': guides/password-mgt/password-policies.md
+        - 'Admin-initiated password reset': guides/password-mgt/forced-password-reset.md
+        - 'Password recovery using email verification': guides/password-mgt/recover-password.md
+        - 'Password recovery using challenge question': guides/password-mgt/challenge-question.md
+        - 'ReCaptcha on invalid password attempts': guides/password-mgt/recaptcha-invalid-password-attempts.md
+        - 'ReCaptcha on invalid challenge question attempt': guides/password-mgt/recaptcha-challenge-question-attempts.md
```
Comment (sherenewso2) [inline en/mkdocs.yml:204]: Title Case

File: en/mkdocs.yml [hunk 19]
```diff
@@ -160,36 +162,67 @@ nav:
...
+          - 'Salesforce': guides/out-prov/outbound-provisioning-with-salesforce.md
+          - 'Role Based Provisioning': guides/out-prov/role-based-provisioning.md
+          - 'Rule Based Provisioning': guides/out-prov/rule-based-provisioning.md
+        - 'Manage Userstores':
+          - 'JDBC as the Primary Userstore': guides/userstore/jdbc-us.md
+          - 'Active Directory as a Secondary Userstore': guides/userstore/ad-us.md
+          - 'LDAP as a Secondary Userstore': guides/userstore/ldap-us.md
+          - 'Plug a Custom Userstore': guides/userstore/custom-us.md
+        - 'Manage Account States':
+          - 'Lock/unlock Accounts': guides/account-states/lock-account.md
```
Comment (sherenewso2) [inline en/mkdocs.yml:195]: unlock --> Unlock

**Pull request merged**